README.md

```diff
@@ -13,31 +13,31 @@ This plugin also provides a way to stop remote desktops that are idle.
 
 With this plugin you can define a `start-command` and `stop-command`.
 Guacamole makes it possible for plugins to listen to the creation and closing of connections.
-Based on this start/stop command are trigged.
+Based on this a start or stop command is trigged.
 
 ### automatic start
 
-If Guacamole try's to connection to a remote desktop. This plugin will check if that remote desktop is running.
+If Guacamole tries to connect to a remote desktop this plugin will check if that remote desktop is running
 and will run `start-command` if that is not the case.
 
 ### automatic stop
 
-This plugin tracks how many connection there are to each host/remote desktop.
+This plugin tracks how many connections there are to each host/remote desktop.
 If the last connection is closed Guacamole will run the stop command for that host.
 
-This plugin also can track if a user is using Guacamole, and can close connection if the user has not moved it's mouse ore used its keyboard for `disconnect-time`
+This plugin can also track if a user is using Guacamole, and can close connection if the user has not moved the mouse ore used the keyboard for `disconnect-time`
```

Comment:
Found a small one you missed. `s/ore/or` ? ;)

Reply:
thanks for merge request

Reply:
This one did not get merged ;)